tests/test_consumer.py

```diff
@@ -1062,3 +1062,43 @@ def test_publisher_id(mock_get_session, consume_fn):
     actual_data = json.loads(
         mock_requests.post.call_args_list[0][1]['data'])
     assert 'msg-example-ci' == actual_data['data'].get('publisher_id'), actual_data
+
+
+def test_topic_namespace_match():
+    fake_msg_path = path.join(json_dir, 'redhat_module_message.json')
+    with open(fake_msg_path) as fake_msg_file:
+        fake_msg = json.load(fake_msg_file)
+
+    fake_msg['topic'] = "/topic/VirtualTopic.eng.ci.baseos-ci.redhat-module.test.complete"
```

Comment:
Could you use single quotes for consistency here and everywhere else in the diff of this file?

Reply:
Sure.